deps/Makefile

```diff
@@ -591,8 +640,11 @@ update-llvm:
 	([ -d "llvm-$(LLVM_VER)/tools/lldb" ] || exit 0;            cd llvm-$(LLVM_VER)/tools/lldb; git pull --ff-only)
 
 ## LIBUV ##
+LIBUV_GIT_URL=git://github.com/JuliaLang/libuv.git
+LIBUV_TAR_URL=https://github.com/JuliaLang/libuv/tarball/$1
+$(eval $(call git-external,libuv,LIBUV,configure,.libs/libuv.a))
```

Comment:
this is supposed to match the `UV_SRC_TARGET` below. does that mean this entire rule is currently broken on msvc?